client/packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -33,6 +35,12 @@ const DetailViewInner = () => {
   const t = useTranslation();
   const { info } = useNotification();
   const isDisabled = useOutbound.utils.isDisabled();
+
+  const { data: prefs } = usePreference(
+    PreferenceKey.SortByVvmStatusThenExpiry,
+    PreferenceKey.ManageVaccinesInDoses
+  );
```

Comment:
Moved the pref query out of modal component, into parent, ensure these prefs are available in time load the lines in the right order etc...#7859